go.mod

```diff
@@ -142,7 +141,7 @@ require (
 	github.com/rancher/steve v0.0.0-20240906202532-e930ae3691b2
 	github.com/rancher/system-upgrade-controller/pkg/apis v0.0.0-20240301001845-4eacc2dabbde
 	github.com/rancher/wrangler v1.1.2
-	github.com/rancher/wrangler/v3 v3.0.0
+	github.com/rancher/wrangler/v3 v3.0.1-rc.1
```

Comment:
Ideally it would be nice not to bring an RC in the same PR but I guess there is no stable version yet that uses this version of gomock.
Does `v3.0.1-rc.1` bring any behavior changes?

Reply:
I believe it's acceptable given the target branch is main